try {
    anotherFn(); 
} catch(error) {
    console.log( 'esto es un error');
}

